WordPress/Classes/Services/BlogService.h

```diff
@@ -58,15 +58,19 @@
 
 - (void)migrateJetpackBlogsToXMLRPCWithCompletion:(void (^)())success;
 
-/*! Syncs an entire blog include posts, pages, comments, options, post formats and categories.
- *  Used for instances where the entire blog should be refreshed or initially downloaded.
- *
- *  \param success Completion block called if the operation was a success
- *  \param failure Completion block called if the operation was a failure
+/**
+ Syncs an entire blog in the background including posts, pages, comments, and blog meta such as
+ post formats, blog options, and categories. Also checks if the blog is multi-author.
+ Used for instances where the entire blog should be refreshed or initially downloaded.
+ */
+- (void)backgroundSyncBlog:(Blog *)blog;
```

Comment:
> What if we just stopped doing this?

I'm a fan of this idea, honestly. A short load time the first time the app is launched shouldn't be a big deal and even if the user happens to have a lot of latency they probably would still be waiting on the initial sync to finish by the time the signed in and tapped to view their post list.

> Also, why the name change? I'm fine with it, but just curious.

I might have been influenced by my perception of how the method is used: fire-and-forget, callbacks never used. If it doesn't make sense I can switch it back.